### ISSUE TITLE
GPU Not working

### ISSUE THREAD

**Issue by HyperSimon** [deepfakes/faceswap#794]
```shell
Loading...
07/15/2019 04:46:55 INFO     Log level set to: INFO
07/15/2019 04:46:56 INFO     Output Directory: C:\Users\LPDR\Desktop\cysd
07/15/2019 04:46:56 INFO     Input Video: C:\Users\LPDR\Desktop\cysd-06.mp4
07/15/2019 04:46:56 INFO     Loading Detect from Mtcnn plugin...
07/15/2019 04:46:56 INFO     Loading Align from Fan plugin...
07/15/2019 04:46:57 INFO     Starting, this may take a while...
07/15/2019 04:46:59 INFO     Initializing Face Alignment Network...
07/15/2019 04:47:00 WARNING  Using CPU
07/15/2019 04:47:06 INFO     Initialized Face Alignment Network.
07/15/2019 04:47:08 INFO     Initializing MTCNN Detector...
07/15/2019 04:47:08 WARNING  From c:\program files\python3\lib\site-packages\tensorflow\python\framework\op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.\nInstructions for updating:\nColocations handled automatically by placer.
07/15/2019 04:47:08 WARNING  From C:\Users\LPDR\faceswap\plugins\extract\detect\mtcnn.py:423: div (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.\nInstructions for updating:\nDeprecated in favor of operator or tf.math.divide.
07/15/2019 04:47:09 WARNING  Using CPU
```

`nvcc -V` :

```shell
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2017 NVIDIA Corporation
Built on Fri_Sep__1_21:08:32_Central_Daylight_Time_2017
Cuda compilation tools, release 9.0, V9.0.176
```

tensorflow@1.13.1

And I install by using `faceswap_setup_x64_v0.99.1.exe` with NVIDIA GPU option.

**Comment by torzdf** [comment]
This most likely an Nvidia driver issue. You must make sure your drivers are installed from the Nvidia website (the Standard Drivers not DCH). Windows basically puts the Machine Learning tools in the wrong place. Unfortunately Windows has made this difficult.

From here: https://www.tenforums.com/graphic-cards/131259-after-win-10-clean-install-cant-install-nvidia-driver-help-2.html#post1619616

> If you want to use standard then remove the current DCH driver using DDU (Display Driver Uninstaller) and install the latest standard driver without an internet connection. Nvidia offers both on their website, 

Make sure you select "Standard" not DCH

**Comment by HyperSimon** [comment]
@torzdf Yes, I first used DDU to uninstall the graphics card driver of the machine, and then downloaded the driver from the official website. The difference between the standard version and DCH is that the former has a driver panel and the latter does not? At present, I have a driver panel.

**Comment by HyperSimon** [comment]
![微信截图_20190715211543](https://user-images.githubusercontent.com/9377192/61219032-e3539b00-a745-11e9-8089-76059b46c85a.png)

**Comment by torzdf** [comment]
It is also where it stores nvidia-smi. It should NOT be in System32 and should be somewhere in C:\Program Files\NVIDIA Corporation\NVSMI

**Comment by HyperSimon** [comment]
![QQ截图20190716125708](https://user-images.githubusercontent.com/9377192/61290776-43f2de80-a7ff-11e9-9043-50debcd2cbb5.png)

You mean this?

**Comment by torzdf** [comment]
Post the output from Tools Menu > Output System Info

**Comment by HyperSimon** [comment]
This is my System info
============ System Information ============
encoding:            cp936
git_branch:          Not Found
git_commits:         Not Found
gpu_cuda:            9.0
gpu_cudnn:           7.6.0
gpu_devices:         GPU_0: GeForce GTX 1060
gpu_devices_active:  GPU_0
gpu_driver:          431.36
gpu_vram:            GPU_0: 6144MB
os_machine:          AMD64
os_platform:         Windows-10-10.0.17763-SP0
os_release:          10
py_command:          .\faceswap.py gui
py_conda_version:    N/A
py_implementation:   CPython
py_version:          3.7.4
py_virtual_env:      False
sys_cores:           12
sys_processor:       Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
sys_ram:             Total: 16230MB, Available: 12164MB, Used: 4066MB, Free: 12164MB

=============== Pip Packages ===============

**Comment by torzdf** [comment]
Ok. This does not tell me what I need to know, specifically what Pip Packages you have installed. The fact that it did not output properly and that it says `py_virtual_env: False` leads me to  believe you are not set up correctly. 

You should be in a virtual environment.

Secondly this:
```From c:\program files\python3\lib\site-packages\tensorflow\python\framework\op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.\nInstructions for updating:\nColocations handled automatically by placer.``` suggests to me you are using Tensorflow 1.13. Tensorflow 1.13 is not compatible with Cuda 9.0 which it says you have on your system. I hugely doubt you have compiled Tensorflow yourself to support Cuda 9.0, given the issues you are having.

Basically you have an unsupported install. You should remove your install and follow the instructions in INSTALL.md. There is no way we can support non-standard installs.

**Comment by HyperSimon** [comment]
@torzdf Thanks for help, I reinstall my requirements by conda, I've been working on this all night. It looks like I'm on the right track. But when I start it, it doesn't seem to work at all. I've been waiting for it for 10 minutes. I know the first start may be a little slow, but is 10 minutes a normal range?

**Comment by torzdf** [comment]
It can happen. You can try killing it and running again. Sometimes Cuda gets stuck caching stuff.

**Comment by HyperSimon** [comment]
My System Info 


============ System Information ============
encoding:            cp936
git_branch:          Not Found
git_commits:         Not Found
gpu_cuda:            9.0
gpu_cudnn:           7.6.0
gpu_devices:         GPU_0: GeForce GTX 1060
gpu_devices_active:  GPU_0
gpu_driver:          431.36
gpu_vram:            GPU_0: 6144MB
os_machine:          AMD64
os_platform:         Windows-10-10.0.17763-SP0
os_release:          10
py_command:          faceswap.py gui
py_conda_version:    conda 4.7.5
py_implementation:   CPython
py_version:          3.6.8
py_virtual_env:      True
sys_cores:           12
sys_processor:       Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
sys_ram:             Total: 16230MB, Available: 9844MB, Used: 6386MB, Free: 9844MB

=============== Pip Packages ===============
absl-py==0.7.1
astor==0.8.0
certifi==2019.6.16
cycler==0.10.0
decorator==4.4.0
fastcluster==1.1.25
ffmpy==0.2.2
gast==0.2.2
grpcio==1.22.0
h5py==2.9.0
imageio==2.5.0
imageio-ffmpeg==0.3.0
joblib==0.13.2
Keras==2.2.4
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.1.0
Markdown==3.1.1
matplotlib==2.2.2
networkx==2.3
numpy==1.16.2
nvidia-ml-py3==7.352.0
opencv-python==4.1.0.25
pathlib==1.0.1
Pillow==6.1.0
protobuf==3.9.0
psutil==5.6.3
pyparsing==2.4.0
python-dateutil==2.8.0
pytz==2019.1
PyWavelets==1.0.3
pywin32==224
PyYAML==5.1.1
scikit-image==0.15.0
scikit-learn==0.21.2
scipy==1.3.0
six==1.12.0
tensorboard==1.12.2
tensorflow-gpu==1.12.0
termcolor==1.1.0
toposort==1.5
tqdm==4.32.2
Werkzeug==0.15.4
wincertstore==0.2

============== Conda Packages ==============
# packages in environment at C:\Users\LPDR\Anaconda3\envs\faceswap:
#
# Name                    Version                   Build  Channel
absl-py                   0.7.1                    pypi_0    pypi
astor                     0.8.0                    pypi_0    pypi
certifi                   2019.6.16                py36_0    defaults
cycler                    0.10.0                   pypi_0    pypi
decorator                 4.4.0                    pypi_0    pypi
fastcluster               1.1.25                   pypi_0    pypi
ffmpy                     0.2.2                    pypi_0    pypi
gast                      0.2.2                    pypi_0    pypi
grpcio                    1.22.0                   pypi_0    pypi
h5py                      2.9.0                    pypi_0    pypi
imageio                   2.5.0                    pypi_0    pypi
imageio-ffmpeg            0.3.0                    pypi_0    pypi
joblib                    0.13.2                   pypi_0    pypi
keras                     2.2.4                    pypi_0    pypi
keras-applications        1.0.8                    pypi_0    pypi
keras-preprocessing       1.1.0                    pypi_0    pypi
kiwisolver                1.1.0                    pypi_0    pypi
markdown                  3.1.1                    pypi_0    pypi
matplotlib                2.2.2                    pypi_0    pypi
networkx                  2.3                      pypi_0    pypi
numpy                     1.16.2                   pypi_0    pypi
nvidia-ml-py3             7.352.0                  pypi_0    pypi
opencv-python             4.1.0.25                 pypi_0    pypi
pathlib                   1.0.1                    pypi_0    pypi
pillow                    6.1.0                    pypi_0    pypi
pip                       19.1.1                   py36_0    defaults
protobuf                  3.9.0                    pypi_0    pypi
psutil                    5.6.3                    pypi_0    pypi
pyparsing                 2.4.0                    pypi_0    pypi
python                    3.6.8                h9f7ef89_7    defaults
python-dateutil           2.8.0                    pypi_0    pypi
pytz                      2019.1                   pypi_0    pypi
pywavelets                1.0.3                    pypi_0    pypi
pywin32                   224                      pypi_0    pypi
pyyaml                    5.1.1                    pypi_0    pypi
scikit-image              0.15.0                   pypi_0    pypi
scikit-learn              0.21.2                   pypi_0    pypi
scipy                     1.3.0                    pypi_0    pypi
setuptools                41.0.1                   py36_0    defaults
six                       1.12.0                   pypi_0    pypi
sqlite                    3.29.0               he774522_0    defaults
tensorboard               1.12.2                   pypi_0    pypi
tensorflow-gpu            1.12.0                   pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
toposort                  1.5                      pypi_0    pypi
tqdm                      4.32.2                   pypi_0    pypi
vc                        14.1                 h0510ff6_4    defaults
vs2015_runtime            14.15.26706          h3a45250_4    defaults
werkzeug                  0.15.4                   pypi_0    pypi
wheel                     0.33.4                   py36_0    defaults
wincertstore              0.2              py36h7fe50ca_0    defaults
